src/config/dynamic-theme-fixes.config

```diff
@@ -28029,6 +28036,35 @@ CSS
 
 ================================
 
+staging.typst.app/universe
```

Comment:
Adding this would need to have it alphabetically sorted not under typst.app/universe

Reply:
However it would be easier to add alias staging.typst.app/universe as a next line under typst.app/universe as it will be the same fixes used and remove this additional copy of fixes.